### PR TITLE
Comment unnecessary code, move reference code

### PR DESCRIPTION
**Pull request by torbsorb**

### CODE REVIEW

File: SurfaceMatchingCreateModelFromFile.hdev [hunk 1]
```diff
@@ -60,12 +60,12 @@
 <l>dev_disp_text (['Position the mouse cursor on the edge of the object', 'Click and hold the left mouse button to begin drawing the ROI around the object', 'Release the left mouse button to end drawing the ROI', 'Click the right mouse button to continue running the program'], 'window', 'bottom', 'left', 'black', [], [])</l>
 <l>draw_region (RegionDraw, Window3D)</l>
 <c></c>
-<c>* Geting min and max Z values encapsulated by the created region</c>
-<l>min_max_gray (RegionDraw, Z, 0, MinZ, MaxZ, RangeZ)</l>
+<c>* Geting min and max Z values encapsulated by the crezated region</c>
```
Comment (SatjaSivcev) [inline SurfaceMatchingCreateModelFromFile.hdev]: cre**z**ated - typo

File: SurfaceMatchingCreateModelFromFile.hdev [hunk 2]
```diff
@@ -60,12 +60,12 @@
...
+<l>* min_max_gray (RegionDraw, Z, 0, MinZ, MaxZ, RangeZ)</l>
 <c></c>
 <c>* Segmenting the Z image with Z values encapsulated by the created region</c>
-<l>threshold (Z, RegionZ, MinZ, MaxZ)</l>
-<l>reduce_domain (Z, RegionZ, Z)</l>
+<l>* threshold (Z, RegionZ, MinZ, MaxZ)</l>
+<l>* reduce_domain (Z, RegionZ, Z)</l>
```
Comment (SatjaSivcev) [inline SurfaceMatchingCreateModelFromFile.hdev]: I think we can remove 63-68. I don't see the point of having it.

**Comment by eskaur** [comment]
Is this PR still relevant? @torbsorb

**Comment by torbsorb** [comment]
Won't fix here and now.

**Pull request closed without merging**